module/VuFind/config/module.config.php

```diff
@@ -828,11 +828,13 @@
     'LibraryCards/ConnectCard', 'LibraryCards/ConnectCardLogin',
     'LibraryCards/DeleteCard',
     'MyResearch/Account', 'MyResearch/ChangeEmail', 'MyResearch/ChangePassword',
-    'MyResearch/CheckedOut', 'MyResearch/CompleteLogin', 'MyResearch/Delete', 'MyResearch/DeleteAccount',
+    'MyResearch/CheckedOut', 'MyResearch/CompleteLogin', 'MyResearch/Delete',
+    'MyResearch/DeleteApiKey', 'MyResearch/DeleteAccount',
```

Comment:
Might as well keep alphabetical order:

```suggestion
    'MyResearch/DeleteAccount', 'MyResearch/DeleteApiKey',
```